modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/topology/api/LogicalTopologyService.java

```diff
@@ -62,6 +62,11 @@ public interface LogicalTopologyService {
      */
     CompletableFuture<LogicalTopologySnapshot> logicalTopologyOnLeader();
 
+    /**
+     * Retrieves the current logical topology snapshot stored in the local storage.
+     */
+    LogicalTopologySnapshot getLogicalTopology();
```

Comment:
Does it make sense to reflect it in the name of the method that the topology is returned based on the local view of the node? Like `localLogicalTopology()`.

Also, `get` prefix does not match the style of `logicalTopologyOnLeader()`: they should both have `get` prefix, or none of them should have it.

Reply:
Agee. I've used the mehod naming from internals, but you proposal looks reasonable.